src/graphql/mutation/checkout/validateLenderEmailForPromo.graphql

```diff
@@ -0,0 +1,9 @@
+mutation validateLenderEmailForPromo($lenderEmailAddress: String!, $managedAccountId: Int!, $promoFundId: Int!) {
+    validateLenderEmailForPromo (
```

Comment:
Does the warning showing here in GH need to be addressed?

Reply:
Yes, I've created a ticket to make sure the new service is available before this code is operationalized...

Reply:
@mcstover have you seen this error before?  I don't really understand what the issue is exactly...  The code is making the appropriate API call it seems so I don't get why the linter is yelling at me.  It's just a warning though, not an error.

  2:5  warning  Cannot query field "validateLenderEmailForPromo" on type "Mutation"  graphql/template-strings